tests/Dockerfile

```diff
@@ -1,10 +1,17 @@
-FROM registry.access.redhat.com/ubi8/ubi:latest
+FROM centos:centos8
 
-RUN dnf install -y bc git go-toolset && \
+ARG CHROMEDRIVER_VER=83.0.4103.39
+
+RUN dnf install -y bc git go-toolset python3-pip unzip && \
     git clone https://github.com/crobby/peak && \    
     cd /peak && \
     git submodule update --init
 
+RUN yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm &&\
+    echo -e "[google-chrome]\nname=google-chrome\nbaseurl=http://dl.google.com/linux/chrome/rpm/stable/x86_64\nenabled=1\ngpgcheck=1\ngpgkey=https://dl.google.com/linux/linux_signing_key.pub" > /etc/yum.repos.d/google-chrome.repo &&\
+    yum -y install google-chrome-stable-83.0.4103.116-1.x86_64 &&\
```

Comment:
FYI - I had to do this because I need to make sure chrome and chrome driver are compatible..so when there is a new major (?) version of chrome, we'd get image build in tests failing and we'd need to update the versions to match again:( Not sure if there is  better way..